src/main.go

```diff
@@ -421,6 +427,30 @@ func (t *tester) Run() error {
 				q.Query = q.Query[:len(q.Query)-1]
 			}
 			t.disconnect(q.Query)
+		case Q_LET:
+			q.Query = strings.TrimSpace(q.Query)
+			eqIdx := strings.Index(q.Query, "=")
+			if eqIdx > 1 {
+				start := 0
+				if q.Query[0] == '$' {
+					start = 1
+				}
+				varName := strings.TrimSpace(q.Query[start:eqIdx])
+				varValue := strings.TrimSpace(q.Query[eqIdx+1:])
+				varSearch := regexp.MustCompile("`(.*)`")
+				varValue = varSearch.ReplaceAllStringFunc(varValue, func(s string) string {
+					s = strings.Trim(s, "`")
+					r, err := t.executeStmtString(s)
+					if err != nil {
+						log.WithFields(log.Fields{
+							"query": s, "line": q.Line},
+						).Error("failed to perform let query")
+						return ""
+					}
+					return r
+				})
```

Comment:
Will this work with something like this from the mysql-test directory in mysql-server repo?
```
t/mysql_binary_mode.test:--let $val0= `SELECT HEX(c1) FROM `$table_name` LIMIT 1`
```

Reply:
No that doesn't work.